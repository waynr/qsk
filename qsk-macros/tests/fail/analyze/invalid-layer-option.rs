use qsk_macros::remap;

fn main() {
    remap!(
        ModLayer[InvalidLayerOption]: {
            F -> F,
        },
    );
}
