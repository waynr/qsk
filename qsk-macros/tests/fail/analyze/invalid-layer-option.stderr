error: invalid layer option

         = help: valid layer options include: ["Active"]

 --> tests/fail/analyze/invalid-layer-option.rs:5:18
  |
5 |         ModLayer[InvalidLayerOption]: {
  |                  ^^^^^^^^^^^^^^^^^^
